thumbor/detectors/face_detector/__init__.py

```diff
@@ -25,7 +25,10 @@ def __add_hair_offset(self, top, height):
         return top
 
     def detect(self, callback):
-        features = self.get_features()
+        try:
+            features = self.get_features()
+        except Exception:
+            features = None
```

Comment:
Sure I could. I'm just not very sure what should be logged/tracked here. The detector is trying to get features within a meta request. To be honest I'm not sure what exactly makes this error. My best guess is that something is not being initialized as a "metadata only" performance optimalization. Any additional thoughts?